src/renderer/components/NewProjectModal/NewProjectView.tsx

```diff
@@ -45,30 +46,42 @@ const NewProjectView = ({
   setProjectName,
 }: Props) => {
   const dispatch = useDispatch();
+  const [isAwaitingProjectName, setIsAwaitingProjectName] =
+    useState<boolean>(true);
 
-  const setProjectInStore = async (project: RuntimeProject) => {
-    dispatch(projectCreated(project));
-  };
+  const setProjectInStore = useCallback(
+    async (project: RuntimeProject) => {
+      dispatch(projectCreated(project));
+    },
+    [dispatch]
+  );
 
-  const handleContinue = async () => {
+  const handleContinue = useCallback(async () => {
     const project = await makeProjectWithoutMedia(projectName);
     if (project === null) {
       return;
     }
     setProjectInStore(project);
     nextView();
-  };
+  }, [nextView, projectName, setProjectInStore]);
+
+  useCustomHook(handleContinue, isAwaitingProjectName);
 
   const handleProjectNameInput = (
     event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
   ) => {
-    setProjectName(event.target.value);
+    setProjectName(event.target.value.trim());
+    if (event.target.value !== '') {
+      setIsAwaitingProjectName(false);
+    } else {
+      setIsAwaitingProjectName(true);
+    }
   };
 
   const continueButton = (
     <PrimaryButton
       onClick={handleContinue}
-      disabled={!projectName.trim()}
+      disabled={isAwaitingProjectName}
```

Comment:
This change brought back the bug mentioned in the comments of [this PR](https://github.com/chloebrett/mlvet/pull/166) - if you go back to the new project modal from the import media modal the continue button is disabled even though the project title field has text in it.

Reply:
Mhmm I see, thank you for bringing that up

Reply:
See this commit: [a7282be](https://github.com/chloebrett/mlvet/pull/175/commits/a7282be8840afa4db73aba0df4af6ca860454241)